docker-unit-test/README.md

```diff
@@ -0,0 +1,40 @@
+# SDK Python
+
+
+### Requerimientos
+
+**MacOS:**
+
+Instalar [Docker](https://docs.docker.com/docker-for-mac/install/), [Docker-compose](https://docs.docker.com/compose/install/#install-compose)
+
+**Windows:**
+
+Instalar [Docker](https://docs.docker.com/docker-for-windows/install/), [Docker-compose](https://docs.docker.com/compose/install/#install-compose)
+
+**Linux:**
+
+Instalar [Docker](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/) y [Docker-compose](https://docs.docker.com/compose/install/#install-compose).
+
+### Como usar
+
+Esta imagin de docker sirve para correr y comprobar la compatibilidad del sdk con distintas versiones de python
+
+**NOTA:** La primera vez que se ejecuta ./docker-build.sh demorará en instalar todo, esperar al menos unos 5 minutos.
```

Comment:
ups, se me paso esto aqui, deberia ser ./build.sh

Reply:
done! :)

Reply:
you were faster than me comment